realue/handler.go

```diff
@@ -160,6 +160,14 @@ func HandleRegCompleteEvent(ue *realuectx.RealUe,
 	stats.LogStats(e)
 
 	m := formUuMessage(common.REG_COMPLETE_EVENT, nasPdu, id)
+	var tmsi string
+	if len(ue.Guti) == 19 {
+		tmsi = ue.Guti[5:]
+	} else {
+		tmsi = ue.Guti[6:]
+	}
+
+	m.Tmsi = tmsi
```

Comment:
5G-S-TMSI field is needed in ServiceRequest packet.
Place these line between line472 and line473, in order to solve #126.

Reply:
Done. Thanks for your feedback.